workflows/prognostic_c48_run/runtime/steppers/reservoir.py

```diff
@@ -153,18 +155,25 @@ def __init__(
         diagnostic_only: bool = False,
         input_averager: Optional[TimeAverageInputs] = None,
         rename_mapping: Optional[NameDict] = None,
+        warm_start: bool = False,
     ):
         self.model = model
         self.synchronize_steps = synchronize_steps
         self.initial_time = init_time
         self.timestep = reservoir_timestep
         self.diagnostic = diagnostic_only
         self.input_averager = input_averager
+        self.warm_start = warm_start
 
         if state_machine is None:
             state_machine = _FiniteStateMachine()
         self._state_machine = state_machine
 
+        if self.warm_start:
+            self.synchronize_steps = 0
```

Comment:
Maybe also raise an error if `synchronize_steps != 0` and is going to be overwritten, just to ensure users know what is happening.